openedx/features/edly/tests/factories.py

```diff
@@ -0,0 +1,50 @@
+"""
+Model factories for unit testing views or models.
+"""
+
+from django.contrib.sites.models import Site
+from factory import SelfAttribute, Sequence, SubFactory
+from factory.django import DjangoModelFactory
+from organizations.tests.factories import OrganizationFactory
+
+from openedx.features.edly.models import EdlyOrganization, EdlySubOrganization
+
+
+class EdlyOrganizationFactory(DjangoModelFactory):
+    """
+    Factory class for Organization model
+    """
+
+    class Meta(object):
+        model = EdlyOrganization
+        django_get_or_create = ('name', 'slug')
+
+    name = Sequence('Edly Organization {}'.format)
+    slug = Sequence('Edly Organization {}'.format)
```

Comment:
This is not a proper slug based on ALPHANUMERIC_VALIDATOR

Reply:
Done